microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -75,8 +91,14 @@ This release includes integration with the greenboot health check framework. Gre
 //[id="microshift-4-13-deprecated-removed-features"]
 //== Deprecated and removed features
 
-//[id="microshift-4-13-bug-fixes"]
-//== Bug fixes
+[id="microshift-4-13-bug-fixes"]
+== Bug fixes
+
+* Previously, the certificate authorities generated `kubeconfig` files for {product-title} embedded components. With this update, the embedded components are reconfigured to ensure the `kubeconfig` files are independent. (link:https://issues.redhat.com/browse/OCPBUGS-10223[*OCPBUGS#10223*])
+
+* Previously, the `systemd-resolved` configuration was used by kubelet as an alternative dns resolver, but the DNS Corefile configuration did not use the same `systemd-resolved` configuration. With this update, {product-title} uses network name resolution provided by `systemd-resolved` for both kublet and cluster DNS when `systemd-resolved.service` is enabled. (link:https://issues.redhat.com/browse/OCPBUGS-6786[*OCPBUGS-6786*])
+
+* Previously, the `microshift-etcd` version was not specified in the output to multiple commands. With this update, `microshift-etcd` outputs the correct version. (link:https://issues.redhat.com/browse/OCPBUGS-8704[*OCPBUGS-8704*])
```

Comment:
I'm not sure this one is worth mentioning. The bug was both introduced and fixed in 4.13.